packages/flutter/lib/src/material/ink_highlight.dart

```diff
@@ -31,118 +27,4 @@ const Duration _kDefaultHighlightFadeDuration = Duration(milliseconds: 200);
 ///    on a [Material].
 ///  * [Ink], a convenience widget for drawing images and other decorations on
 ///    Material widgets.
-class InkHighlight extends InteractiveInkFeature {
```

Comment:
Once `InkHighlight` is migrated out of the Material library, it could be used to create the Cupertino touch effects as shown [here](https://github.com/flutter/flutter/pull/149963#issuecomment-2224122127).